### PR TITLE
Sending packets with increasing timestamp according to Akamai docs

### PR DESCRIPTION
**Pull request by marcin-adamczewski**
Temp PR to demonstrate sample implementation of sending frames with increasing timestamp

### CODE REVIEW

**Comment by marcin-adamczewski** [comment]
In short, the algorithm works as follows:
Collect all frames until the next video and audio frame has larger timestamp than some current pivotFrame. Then send all the frames sorted by timestamp and assign a new pivotFrame.

Basically, in order for it to work both audio and video frames have to be sent regularly as I wait for both audio and video frame dts to be larger than pivot frame dts before sending anything out. I thought it will be problematic when audio is muted, but you send "empty" audio frames in that case. So it works fine when audio is muted. Is there any case when only video or audio frames would be sent?

I also removed assigning `mAudioSequenceHeader` and `mVideoSequenceHeader` as you don't use it anywhere.

BTW I've tested this on Vimeo as well which doesn't require such timestamp sorting and it works fine too.

**Comment by pedroSG94** [comment]
I don't understand. What do you means with "worked for a while"?
Do you means your implementation or with my implementation?
Where is the error? Stream stop after few minutes?

**Comment by marcin-adamczewski** [comment]
@pedroSG94 Sorry for providing that little info :)
I've tested your solution again. The result is that either there is video but no audio or the stream works fine for 2 seconds and stops (like it was before applying changes).

I've logged `dts` values in SrsFlvMuxer.sendFlvTag() method. It turned out that subsequent frames didn't have increasing `dts`. Correct me if I'm wrong, but It's probably because 
`frame.dts = (int)((System.nanoTime() / 1000 - startTs) / 1000);`
was added in `StsFlvMuxer.writeRtmpPacket()` method which can be called from either Video or Audio encoder, which works on different threads.
After moving this line `frame.dts = (int)((System.nanoTime() / 1000 - startTs) / 1000);`
to SrsFlvMuxer.sendFlvTag() method, which is right before sending a frame, the dts values are increasing and stream works well.
So basically your simple solution works very well, we'd just need to move setting `dts` just before sending a frame.

**Comment by pedroSG94** [comment]
You are right but that was working for me for some reason O.O (maybe only luck)
This should be like this:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/ad6eccb392f37ada61be354cd55fbbf18a1ec2e8
Also I added a method to make it optional

**Comment by marcin-adamczewski** [comment]
@pedroSG94 Works well. Good idea with making it optional, however, there is no access to SrsFlvMuxer class from RtmpCamera class so we can't set this flag. 

What do you think about adding a comment above the `akamaiTs` flag to make it clear what is it for, like:
> Some Livestream hosts use Akamai auth that requires RTMP packets to be sent with increasing timestamp order regardless of packet type. More info here https://learn.akamai.com/en-us/webhelp/media-services-live/media-services-live-encoder-compatibility-testing-and-qualification-guide-v4.0/GUID-F941C88B-9128-4BF4-A81B-C2E5CFD35BBF.html

**Comment by pedroSG94** [comment]
Yes, good idea.
I will provide access to it and set that comment in the method.

**Comment by pedroSG94** [comment]
I merged this feature to master. Thanks for your help

**Comment by marcin-adamczewski** [comment]
Thank you :)
